lib/error_tracker.ex

```diff
@@ -119,15 +119,14 @@ defmodule ErrorTracker do
     {:ok, stacktrace} = ErrorTracker.Stacktrace.new(stacktrace)
     {:ok, error} = Error.new(kind, reason, stacktrace)
     context = Map.merge(get_context(), given_context)
-
-    context =
-      if bread_crumbs = bread_crumbs(exception),
-        do: Map.put(context, "bread_crumbs", bread_crumbs),
-        else: context
+    breadcrumbs = exception_breadcrumbs(exception) ++ get_breadcrumbs()
```

Comment:
I am not sure about this. Let's say we have the following situation:

1. User adds breadcrumb A
2. User adds breadcrumb B
3. Exception is reported which includes breadcrumbs [C, D]

I would expect to have the following breadcrumbs [A, B, C, D], but instead we would have [C, D, A, B].

The problem is that we don't have a way to know if a manual breadcrumb happened before or after an exception breadcrumb, so there may not be a proper solution for this if we allow setting manual breadcrumbs.